lib/Scheduler/sched_basic_data.cpp

```diff
@@ -698,12 +698,13 @@ bool SchedInstruction::ProbeScsrsCrntLwrBounds(InstCount cycle) {
 void SchedInstruction::ComputeAdjustedUseCnt_() {
   Register **uses;
   int useCnt = GetUses(uses);
+  int tempAdjustedUseCnt = useCnt;
```

Comment:
NITPICK: You could just write:
```suggestion
  adjustedUseCnt_ = GetUses(uses);;
```